src/component/snack/snack.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```

Comment:
First, correct the name of the file itself according to PascalCase. Second, don't import React